src/components/ToggleInput/ToggleInput.tsx

```diff
@@ -1,27 +1,42 @@
 import { InputTag } from "components/InputTag/InputTag";
 import { ToggleSwitch } from "components/ToggleSwitch/ToggleSwitch";
 import { useToggle } from "hooks/useToggle";
-import { DetailedHTMLProps, InputHTMLAttributes, useEffect, useState } from "react";
+import { DetailedHTMLProps, InputHTMLAttributes, useEffect } from "react";
 import style from "./ToggleInput.module.scss";
 
 type Props = {
-    label: string,
-    onToggle?: (state: boolean) => void,
-} & DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>
+    id: string;
+    onChange: (state: number) => void;
+    onToggle: (state: boolean) => void;
+    disabled: boolean;
```

Comment:
Removed disabled from DetailedHTMLProps because it might be being overwritten by that type (changing the type from boolean to boolean | undefined)